packages/react/src/components/DatePickerInput/DatePickerInput.js

```diff
@@ -31,6 +31,11 @@ export default class DatePickerInput extends Component {
      */
     labelText: PropTypes.node.isRequired,
 
+    /**
+     * Specify the size of the Text Input. Currently supports either `sm` or `xl` as an option.
+     */
+    size: PropTypes.oneOf(['sm', 'xl']),
```

Comment:
```suggestion
    /**
     * Specify the size of the Date Picker. Currently supports either `sm` or `xl` as an option.
     */
    size: PropTypes.oneOf(['sm', 'xl', '']),
```

Reply:
Same comment as in the other PR --
Per this conversation https://github.com/carbon-design-system/carbon/pull/5472#issuecomment-592221372, it may be best to have a default of `lg` instead? Given that's the intended true default for all of these sizes? 🤔

Reply:
@jendowns okay now I'm getting so confused haha. Sorry for being dense here.

It seems like `lg` doesn't have a class associated with it, so it would make sense for `size` to default to nothing, right? Instead of `lg` as that would create a class name that would essentially noop? Let me know if I'm being dumb here lol.

Reply:
Yeah I'm also a bit confused on what went on while I was asleep 😄 

I think it's best to do something like you suggested here, updating the PropTypes to include `` as valid, and then setting that as the default value. Since there are no styles associated with `--lg`, I'd rather not output an extra class.

Reply:
It also seems like we can just remove the `static defaultProps = {  size: ' ' }` and it doesn't cause any errors, without needing to add `size: PropTypes.oneOf(['sm', 'xl', '']),`

Reply:
Thanks all!
@tw15egan the short version is that `TextInput` and `Select` has propType errors generated in `master`, because `size=""` by default but an empty string was not allowed in the propTypes.

But it looks like your other PR, removing that default value altogether, fix that. 👍 
Personally I prefer always having a true default for something but I'm totally cool with removing the default in this case. Either way, the console errors are gone. 🎉